.github/workflows/main.yml

```diff
@@ -1,40 +1,75 @@
-# This is a basic workflow to help you get started with Actions
+---
+name: Run Tests
 
-name: CI
-
-# Controls when the action will run.
 on:
-  # Triggers the workflow on push or pull request events but only for the main branch
   push:
     branches: [ main, 'v*']
   pull_request:
     branches: [ main, 'v*' ]
+  release:
+    types: [released]
 
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+      - uses: pre-commit/action@v3.0.0
+        with:
+          extra_args: -a
+
   tests:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        django-version: ["3.2.0", "4.1.3"]
```

Comment:
I just found out that there is no other place that uses `django-version`, perhaps we need a separate install step to install it?